package.json

```diff
@@ -1,28 +1,33 @@
 {
   "name": "wiki-table-editor",
   "main": "dist/wiki-table-editor.js",
-  "version": "1.1.4",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
I don't think NPM will let us publish this if there's a `private: true` in the package.json. The version should be 1.1.5 to follow semver too.